lib/workers/repository/dependency-dashboard.ts

```diff
@@ -312,7 +314,10 @@ export async function ensureDependencyDashboard(
   }
 
   if (config.dependencyDashboardFooter?.length) {
```

Comment:
This should use `is.string` instead

Reply:
This is not modified code!!!
I couldn't recode all !